Strategy11/business-directory-plugin
# Make the prompt to upgrade dismissible

### Issue link:
https://github.com/Strategy11/business-directory-premium/issues/103

### Changes proposed in this PR:
With this PR, we will make the Download Premium notice dismissible.

### How to test the changes in this Pull Request:
- Active one BD add-on without activating the Premium version.
- You need to dismiss the `"You have modules installed, but Business Directory Premium is missing. Install this plugin for extra features and easy license management."` notice in the BD admin panel and another page in wp-admin, like the Plugins page management.
- In both cases, this message should not be displayed after dismissing you.
